packages/dart_frog_cli/lib/src/dev_server_runner/restorable_directory_generator_target.dart

```diff
@@ -0,0 +1,112 @@
+import 'dart:collection';
```

Comment:
The code in this file is identical to a subsection of what we had on `packages/dart_frog_cli/lib/src/commands/dev/dev.dart`